Expose new Enum values for VirtualNetworkGateway sizes.

ritwikbasu opened the pull request:
Add new enum values to VirtualNetworkGateway tier and name enums.

ritwikbasu committed:
Expose new Enum values for VirtualNetworkGateway sizes.

Add new enum values to VirtualNetworkGateway tier and name enums.

msftclas commented:
@ritwikbasu,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

salameer commented on arm-network/2017-03-01/swagger/virtualNetworkGateway.json:
```diff
@@ -1156,12 +1156,15 @@
         "name": {
           "type": "string",
-          "description": "Gateway SKU name. Possible values are: 'Basic', 'HighPerformance','Standard', and 'UltraPerformance'.",
+          "description": "Gateway SKU name. Possible values are: 'Basic', 'HighPerformance','Standard', 'UltraPerformance', 'VpnGw1', 'VpnGw2' and 'VpnGw3'.",
```
Hi I would recommend, removing the possible values in the description and it would be auto generated from the documentation tools

salameer commented on arm-network/2017-03-01/swagger/virtualNetworkGateway.json:
```diff
@@ -1170,12 +1173,15 @@
         "tier": {
           "type": "string",
-          "description": "Gateway SKU tier. Possible values are: 'Basic', 'HighPerformance','Standard', and 'UltraPerformance'.",
+          "description": "Gateway SKU tier. Possible values are: 'Basic', 'HighPerformance','Standard', 'UltraPerformance', 'VpnGw1', 'VpnGw2' and 'VpnGw3'.",
```
Hi I would recommend, removing the possible values in the description and it would be auto generated from the documentation tools

salameer reviewed (changes requested):
Hi I would recommend, removing the possible values in the description and it would be auto generated from the documentation tools

salameer commented:
The rest looks good.

salameer commented:
However note that there are issues that I detected that are not related to your changes, I'll be logging them and assigning them to you to fix or escalate to the right party on your side

ritwikbasu committed:
Incorporating review comments.

https://github.com/Azure/azure-rest-api-specs/pull/1130 - incorporating
review comments.

salameer commented:
Changes approved, merging

AutorestCI commented:
No modification for NodeJS

AutorestCI commented:
[![Build Status](https://travis-ci.org/AutorestCI/azure-sdk-for-python.svg?branch=RestAPI-PR1130)](https://travis-ci.org/AutorestCI/azure-sdk-for-python) https://github.com/Azure/azure-sdk-for-python/pull/1092

AutorestCI commented:
[![Build Status](https://travis-ci.org/AutorestCI/azure-sdk-for-ruby.svg?branch=RestAPI-PR1130)](https://travis-ci.org/AutorestCI/azure-sdk-for-ruby) https://github.com/Azure/azure-sdk-for-ruby/pull/758
